### DISCUSSION TITLE
How to disable row clicks?

### DISCUSSION THREAD

**Discussion by RealityGamin** [laravel/nova-issues#4390] [Q&A] [answered]
- Nova Version: 4.7

### Description:

How is it possible to disable the row click event? there is already a "view button" and sometimes users want to copy text from the index page, but whenever clicked on the index somewhere to copy, it will open, anyway to disable that?

**Comment by dmason30** [comment]
You can make Text fields copyable to avoid this issue. 
https://nova.laravel.com/docs/4.0/resources/fields.html#copying-text-field-values-to-the-clipboard

There is no built-in way to disable it. There maybe some way to override it in vue but not sure.

**Comment by m-lotze** [comment]
There is a package for configuring the index row click:

https://github.com/marshmallow-packages/nova-resource-click

Although this imo should be in core.

**Comment by ss071109** [comment]
@m-lotze 

https://nova.laravel.com/docs/4.0/resources/#resource-table-click-action

We can set $clickAction to ignore in each resource to disable index row click

File path : /vendor/laravel/nova/src/Resource.php:197

>  /**
     * The click action to use when clicking on the resource in the table.
     * Can be one of: 'detail' (default), 'edit', 'select', 'preview', or 'ignore'.
     *
     * @var string
     */
    public static $clickAction = 'detail';

    
Hope it can help someone

**Reply by harisnodesol** [reply]
it worked actually thanks

**Comment by davidhemphill** [comment, marked as answer]
Set the `clickAction` property on the base `Resource` Nova publishes during installation:

```php
<?php

namespace App\Nova;

use Laravel\Nova\Http\Requests\NovaRequest;
use Laravel\Nova\Resource as NovaResource;

abstract class Resource extends NovaResource
{

    /**
     * The click action to use when clicking on the resource in the table.
     * Can be one of: 'detail' (default), 'edit', 'select', 'preview', or 'ignore'.
     *
     * @var string
     */
    public static $clickAction = 'ignore';

    // ...
}
```

**Reply by sfinktah** [reply]
It would be nice if the (emulated) click functionality -- I assume it's a mousedown & mouseup event handler -- actually emulated a click event more accurately.

e.g., filter events that have too much mouse movement or elapsed time between down & up.

I don't mind the functionality, it just really starts to grate when you are selecting text, and it thinks it's a click.
